core/src/schema.jl

```diff
@@ -58,22 +56,6 @@ function nodetype(
     return Symbol(node[begin:length(n)]), k
 end
 
-@version NodeV1 begin
-    fid::Int
-    name::String = isnothing(s) ? "" : String(s)
-    type::String = in(Symbol(type), nodetypes) ? type : error("Unknown node type $type")
-    allocation_network_id::Union{Missing, Int}
-end
-
-@version EdgeV1 begin
-    fid::Int
-    name::String = isnothing(s) ? "" : String(s)
-    from_node_id::Int
-    to_node_id::Int
-    edge_type::String
-    allocation_network_id::Union{Missing, Int}
-end
-
```

Comment:
These schemas were not used in the core, and their translations in Python were also not used, they are handwritten elsewhere. So best to just remove.